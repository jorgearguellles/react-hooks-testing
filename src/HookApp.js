import React from 'react'

export const HookApp = () => {
    return (
        <div>
            <h1>Hello Sexy</h1>
        </div>
    )
}
