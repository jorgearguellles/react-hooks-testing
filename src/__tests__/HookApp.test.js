import React from 'react';
import { shallow } from 'enzyme';
import { HookApp } from '../HookApp.js';
import '@testing-library/jest-dom';


describe('Tests in <HookApp.ja />', ()=>{
  test('1. Snapshot correctly', ()=>{
    const wrapper = shallow( <HookApp/> );
    expect(wrapper).toMatchSnapshot();
  })
})