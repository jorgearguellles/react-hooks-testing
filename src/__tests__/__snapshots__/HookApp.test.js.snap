// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tests in <HookApp.ja /> 1. Snapshot correctly 1`] = `
<div>
  <h1>
    Hello Sexy
  </h1>
</div>
`;
